--- pipewalker-0.8.1/extra/pipewalker.desktop~	2010-04-30 08:41:52.000000000 +0200
+++ pipewalker-0.8.1/extra/pipewalker.desktop	2010-05-04 10:20:49.576766025 +0200
@@ -1,12 +1,11 @@
 [Desktop Entry]
-Version=1.0
 Type=Application
 Name=PipeWalker
 Comment=Connect all hosts to network!
+Comment[pl]=Połącz wszystkie komputery do sieci!
 Comment[ru]=Объедините все компьютеры в сеть!
 Comment[fr]=Connectez tous les ordinateurs au réseau!
-TryExec=pipewalker
 Exec=pipewalker
 Categories=Game;LogicGame;
-Icon=pipewalker.xpm
-MimeType=
+Icon=pipewalker
+# vi: encoding=utf-8
